core/web/evm_transfer_controller_test.go

```diff
@@ -279,21 +294,90 @@ func TestTransfersController_CreateSuccess_eip1559(t *testing.T) {
 	amount, err := assets.NewEthValueS("100")
 	require.NoError(t, err)
 
+	timeout := 5 * time.Second
 	request := models.SendEtherRequest{
 		DestinationAddress: common.HexToAddress("0xFA01FA015C8A5332987319823728982379128371"),
 		FromAddress:        key.Address,
 		Amount:             amount,
+		WaitAttemptTimeout: &timeout,
 	}
 
 	body, err := json.Marshal(&request)
 	assert.NoError(t, err)
 
 	resp, cleanup := client.Post("/v2/transfers", bytes.NewBuffer(body))
-	t.Cleanup(cleanup)
+	defer t.Cleanup(cleanup)
```

Comment:
Why defer here? Should be either-or, not both, no?
```suggestion
	t.Cleanup(cleanup)
```

Reply:
ah good catch - that's leftover from trying to figure out why the db triggers weren't working. will remove